src/content/blog/filecoin-ecosystem-teams-unveil-l2s.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Filecoin Ecosystem Teams Unveil L2s
+created-on: 2024-11-11T05:05:00.000Z
+updated-on: 2024-11-11T05:05:00.000Z
+published-on: 2024-11-11T05:05:00.000Z
+category: news
+description: Akave and Storacha Share Announcements at FIL Bangkok Signaling New
+  Era for the Filecoin Network
+image:
+  src: /assets/images/image.webp
+seo:
+  twitter:
+    card: summary
+    site: "@filfoundation"
+  title: Filecoin Ecosystem Teams Unveil L2s
```

Comment:
@dcassagnol @ianconsolata Per hint, SEO title can be left out if it's the same a main title.